### DISCUSSION TITLE
Welcome to WebLINX Discussions!

### DISCUSSION THREAD

**Discussion by xhluca** [McGill-NLP/weblinx#3] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by TianWilliam0** [comment]
Yeah, are there any code for a simple usage? 
For example, I want to download some papers of AI in https://www.connectedpapers.com/  , how can I write the codes?
Thanks

**Reply by xhluca** [reply]
@JYAZZX we currently do not have a platform for executing the predictions made by an action model. to do that, you will need to use playwright or selenium, using the parsed json (see `weblinx.processing.outputs`). For running the models and processing the inputs, please take a look at the `modeling/` subdir.

**Reply by TianWilliam0** [reply]
@xhluca thank you for your reply!
 As I understand it, Weblinx can only predict actions, not direct clicks right?
 If I need to perform realistic operations, I can only utilize selenium?
 If I need to predict other sites, do I need to pass the data into a database for training? 
 Looking forward to your reply!

**Reply by xhluca** [reply]
Yes a tool like Selenium is needed. For new sites you can use weblinx models as is or finetune them if you wish.

**Reply by TianWilliam0** [reply]
That's all! Thank you for you reply!

**Reply by xhluca** [reply]
Hope you enjoy using Weblinx!
